ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -56,27 +56,31 @@
   public static final short UNKNOWN_CONTAINER_ID = -1;
 
   /**
-   * The id of {@link #UNKNOWN_PUBLIC_CONTAINER}.
+   * The id for the containers to be associated with the blobs that are put without specifying a target container,
+   * but are specified public. {@link #UNKNOWN_PUBLIC_CONTAINER} is one of the containers that use it.
    */
-  public static final short UNKNOWN_PUBLIC_CONTAINER_ID = -2;
+  public static final short UNKNOWN_PUBLIC_CONTAINER_ID = 0;
```

Comment:
this need not be called "unknown_" anymore - this the default public container id - even for valid accounts. Correct?